### PR TITLE
Updated composite views docs

### PR DESCRIPTION
**Pull request by umbreak**
screencapture-file-Users-wajerowi-code-nexus-docs-target-site-docs-delta-api-current-views-composite-view-api-html-2021-05-18-09_36_20-2

screencapture-file-Users-wajerowi-code-nexus-docs-target-site-docs-delta-api-current-views-composite-view-api-html-2021-05-18-09_36_20

### CODE REVIEW

**Review by bogdanromanx — APPROVED** [review]

File: docs/src/main/paradox/docs/delta/api/current/assets/views/composite/fetched.json [hunk 1]
```diff
@@ -0,0 +1 @@
...http://music.com/source_albums","@type":"CrossProjectEventStream","identities":[{"@id":"http://localhost:8080/v1/anonymous","@type":"Anonymous"}],"includeDeprecated":false,"project":"myorg/albums","resourceSchemas":[],"resourceTypes":["http://music.com/Album"],"_uuid":"eccca1d3-3044-4494-bde6-6c79f56fa649"},{"@id":"http://music.com/source_songs","@type":"RemoteProjectEventStream","endpoint":"http://localhost:8080/v1","includeDeprecated":false,"project":"myorg/songs","resourceSchemas":[],"resourceTypes":["http://music.com/Song"],"_uuid":"aa788720-8e0c-4504-8d54-24c597e261e5"}],"_constrainedBy":"https://bluebrain.github.io/nexus/schemas/views.json","_createdAt":"2021-05-17T14:43:41.763Z","_createdBy":"http://localhost:8080/v1/anonymous","_deprecated":false,"_incoming":"http://localhost:8080/v1/views/myorg/bands/composite_view/incoming","_outgoing":"http://localhost:8080/v1/views/myorg/bands/composite_view/outgoing","_project":"http://localhost:8080/v1/projects/myorg/bands","_rev":3,"_self":"http://localhost:8080/v1/views/myorg/bands/composite_view","_updatedAt":"2021-05-17T14:58:28.483Z","_updatedBy":"http://localhost:8080/v1/anonymous","_uuid":"4b6fe5bf-675c-40bf-88ac-2b26575236c9"}
```
Comment (wwajerowicz) [inline docs/src/main/paradox/docs/delta/api/current/assets/views/composite/fetched.json:1]: this file could use better formatting.

File: docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ where...
 
 ### CrossProjectEventStream
 
-This source reads events in a streaming fashion from the defined project event log in the current Nexus deployment.
+This source reads events in a streaming fashion from the defined project event log in the current Nexus Delta deployment.
 
-The specified list of identities will be used to retrieve the resources from the project. The target project must have `resources/read` permissions in order to read events.
+The specified list of identities will be used to retrieve the resources from the project. The target project must have `events/read` permissions in order to read events.
```
Comment (wwajerowicz) [inline docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md:52]: The identities must have `events/read` permission on the target project ....

File: docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md [hunk 3]
```diff
@@ -249,259 +256,354 @@ The following example creates a Composite view containing 3 sources and 2 projec
...
   {...}
 ```
 
-The supported payload is defined on the 
-@link:[ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html){ open=new }
+... where `{previous_rev}` is the last known revision number for the view.
+
+@@@ note { .warning }
 
+Updating a view creates a new Elasticsearch index and deletes the existing one. The indexing process will start from the
```
Comment (wwajerowicz) [inline docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md:412]: It creates a new index/namespace for all the projection, not only for Elasticsearch.

File: docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md [hunk 4]
```diff
@@ -249,259 +256,354 @@ The following example creates a Composite view containing 3 sources and 2 projec
...
+
+Payload
+:   @@snip [tag.json](../assets/tag.json)
+
+Response
+:   @@snip [tagged.json](../assets/views/composite/tagged.json)
+
+## Deprecate
+
+Locks the view, so no further operations can be performed. It also stops indexing any more resources into it and deletes the underlying index.
```
Comment (wwajerowicz) [inline docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md:461]: and deletes all the underlying Elasticsearch indices and Blazegraph namespaces.

File: docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md [hunk 5]
```diff
@@ -249,259 +256,354 @@ The following example creates a Composite view containing 3 sources and 2 projec
...
+
+Deprecating a view is considered to be an update as well.
+
+@@@ note { .warning }
+
+Deprecating a view deletes the view index, making the view not searchable.
```
Comment (wwajerowicz) [inline docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md:467]: Same here:  deletes all the underlying Elasticsearch indices and Blazegraph namespaces.

**Review by wwajerowicz — CHANGES_REQUESTED** [review]

**Review by wwajerowicz — APPROVED** [review]

**Review by bogdanromanx — APPROVED** [review]

**Review by imsdu — APPROVED** [review]

**Pull request merged**